charts/graphscope-store/templates/_helpers.tpl

```diff
@@ -75,7 +75,7 @@ Return the proper graphscope-store image name
 
 {{/*
 Return the proper image name
-{{ include "graphscope-store.images.image" ( dict "imageRoot" .Values.path.to.the.image "global" $ "DefaultTag" .Chart.AppVersion ) }}
+{{ include "graphscope-store.images.image" ( dict "imageRoot" .Values.path.to.the.image "global" $ "DefaultTag" .DefaultTag ) }}
```

Comment:
Delete the `"DefaultTag" . DefaultTag` as you won't use `imageRoot.DefaultTag` anymore?

Reply:
The `DefaultTag` is a new KV parameter pointing to `.Chart.AppVersion` and is used in the next logic.